doc/source/whatsnew/v0.22.0.txt

```diff
@@ -27,6 +27,17 @@ Other Enhancements
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_index()`` to determine whether the index will be rendered in ouptut (:issue:`14194`)
 - :class:`pandas.io.formats.style.Styler` now has method ``hide_columns()`` to determine whether columns will be hidden in output (:issue:`14194`)
 
```

Comment:
can you add a ref here

Reply:
I'm sorry, not used to work with sphinx. Do you mean something like this here:

```
- :func:`get_dummies` now supports ``dtype`` argument, see :ref:`here <whatsnew_0220.enhancements.get_dummies_dtype>` for more  (:issue: `18330`)
```

and then this before the actual description block:
```
.. _whatsnew_0220.enhancements.get_dummies_dtype
```
?